internal/tiltfile/tiltfile_state.go

```diff
@@ -654,7 +651,14 @@ func (s *tiltfileState) assembleK8sV1() error {
 }
 
 func (s *tiltfileState) assembleK8sV2() error {
-	err := s.assembleK8sByWorkload()
+	//Note: We're using UniqueNames() over here just to check for duplicate resources, hence
```

Comment:
```suggestion
	//Note: We're using UniqueNames() over here just to check for duplicate YAML entities, hence
```

Reply:
Aaaaa I keep making this mistake of referring to YAML entities as resources haha